diff --git a/c_src/MySensors/hal/architecture/Linux/MyMainLinux.cpp b/c_src/MySensors/hal/architecture/Linux/MyMainLinux.cpp
index f67ea2f..fc0444b 100644
--- a/c_src/MySensors/hal/architecture/Linux/MyMainLinux.cpp
+++ b/c_src/MySensors/hal/architecture/Linux/MyMainLinux.cpp
@@ -412,7 +412,7 @@ int main(int argc, char *argv[])
 	}

 	log_opts = LOG_CONS;
-	if (foreground && isatty(STDIN_FILENO)) {
+	if (foreground) {
 		// Also print syslog to stderror
 		log_opts |= LOG_PERROR;
 	}
